Docs: monthly reports (#25964)
grafana/grafana: docs/sources/enterprise/reporting.md

* **Time -** Time of day in 24 hours format when the report should be sent.
* **Time zone -** Time zone for the **Time** field.

#### Monthly

> Only available in Grafana Enterprise v7.1+.

Monthly reports are generated once per month. All fields are required.

> * **Day in month -** Day of the month the when report should be sent. (it's possible to select `last` for reports that should go out on the last day of the month).

Reviewer: ```suggestion
* **Day in month -** Day of the month when the report should be sent (it's possible to select `last` for reports that should go out on the last day of the month).
```
Just noticed that I rewrote all dayInMonth variables to dayOfMonth and forgot to rename the field. I think it should be "Day of month" or "Day of the month"?

Reviewer 2: If it were regular writing, it would be "day of the month", but for variables, shorter seems better.

Reviewer 2: Remove the parenthesis and replace "it's possible to" with "You can".

Author: Thank you, I will incorporate the feeback!

> * **Time -** Time of day in 24 hours format when the report should be sent.

Reviewer 2: ```suggestion
* **Time -** Time the report is sent, in 24-hour format.
```

Author: Thank you, I will incorporate the feeback!
